test/e2e/main/custom_configuration_test.go

```diff
@@ -31,6 +31,13 @@ func TestUserJsonCustomConfig(t *testing.T) {
 	testCustomConfig(t, configMap)
 }
 
+func TestUserLog4jCustomConfig(t *testing.T) {
+	t.Parallel()
+	configMap := newCustomConfigMap(t.Name(), "xml")
+	configMap = addLog4jToCustomConfigMap(configMap)
+	testCustomConfig(t, configMap)
```

Comment:
Instead of adding a new test, I would update `testCustomConfig` so that the custom xml is added for all tests. We can then update `testCustomConfig` so that we retrieve the logs of the server pod and check to see if there's a TRACE statement to ensure that the custom logging has been applied as expected.

Reply:
I like the idea, will update the test as commented.